test/cli.spec.js

```diff
@@ -1,18 +1,28 @@
+import path from "path";
```

Comment:
Can we use the `node:` protocol in this import?

```suggestion
import path from "node:path";
```

Reply:
In the `fs` import too?

Reply:
Also, note that this is not consistent with the rest of `import path` statements of the project.

Reply:
> Also, note that this is not consistent with the rest of import path statements of the project.

The intent of the comment was to do it only for tests as I thought that some of the Node.js version SVGLint should support don't support `node:` protocol imports. However the CI tests proved me wrong 😅 


> In the `fs` import too?

Yes to all imports for builtin modules ([ref](https://nodejs.org/api/esm.html#node-imports)).

Reply:
IMHO, I think that this rewriting task should be in a separate PR. I've reverted the change.

Reply:
Agreed :+1: thought it would be limited to this file.